nix/haskell-project.nix

```diff
@@ -38,6 +38,19 @@ in
         in
         lib.mapAttrs build-haskell-package config.packages;
 
+      # TODO: move this to `parser.nix` and add tests
+      parseExecutables = pkg:
+        let
```

Comment:
- Rename `find-haskell-paths/` to `nix-haskell-parsers`
- Have `nix-haskell-parsers/default.nix` return a attrset of two functions, including this one.

Reply:
I have named it haskell-parsers as it is already under the directory nix